ShopLens/ShopLensForms/ShopLens.cs

```diff
@@ -57,21 +57,26 @@ private void CommandRecognized_WhatIsThis(object sender, EventArgs e)
         {
             if (live_video.Image != null)
             {
-                Bitmap image = (Bitmap)live_video.Image.Clone();
-                capture_picture.Image = (Bitmap)live_video.Image.Clone();
+                var image = (Image)live_video.Image.Clone();
```

Comment:
I thought we agreed to comment out the line `capture_picture.Image = image`? Did changing Bitmap to Image fix that crash?

Reply:
No it didn't fix the crash, it's just that I made this branch from my voice recognizer interface branch when I was trying to fix the thread issue and I kinda left it there uncommented..